src/examples/regression_test.json

```diff
@@ -164,5 +164,18 @@
       "data": "",
       "hash": ""
     }
+  },
+  "Bitwise Primitive": {
+    "digest": "Bitwise Primitive",
+    "methods": {
+      "xor": {
+        "rows": 15,
```

Comment:
15 seems correct to me:
* we can do 16 bits per xor gate
* so it's `length/16` xor gates, plus 1 zero gate
* since the circuit does 16, 32, 48, 64, we get 1+1, 2+1, 3+1, 4+1 which sums to 14
* the 15th row is caused by encoding the 0 constant (we use `assertEquals(0)` multiple times)

Reply:
Does the above means that this primitive performs all of the 4 xor variants together? Instead of one per variant (one Xor for 16 of 3 rows, another for 64 with 6 rows, etc). Or is this circuit just being used for tests?